scripts/generate-marko-module-map.js

```diff
@@ -7,9 +8,9 @@ const readFileAsync = promisify(fs.readFile);
 const writeFileAsync = promisify(fs.writeFile);
 const fileExistsAsync = promisify(fs.exists);
 const NODE_MODULES_PATH = './node_modules';
-const MARKO_DIST = 'marko/dist';
+const MARKO_DIST = `marko/${isDebug ? 'src' : 'dist'}`;
 const BASE_PATH = `${NODE_MODULES_PATH}/${MARKO_DIST}`;
-const OUTPUT_PATH = './src/marko-modules-mocking-map.json';
```

Comment:
Is there any reason to remove `src` form the output path? 

Marko-jest test utils will pick the JSON file from src folder (see https://github.com/abiyasa/marko-jest/blob/master/src/test-utils.js#L5) so it won't pick up the new generated JSON